### PR TITLE
Refactorization of pipeline 2T6S & Pipeline runner

### PR DESCRIPTION
**Pull request by bclenet**
1. [REFAC] :paintbrush: `PipelineTeam2T6S` is a class inheriting from `Pipeline` that implements the work of team 2T6S
    - :microscope: non regression tests have been done, showing no differences between the 2nd level con* maps computed from this implementation and those computed with the previous one.
    - :microscope: unit testing added in :file_folder: `tests`

2. [SRC] :runner: `PipelineRunner` , an app to run pipelines in a convenient way, was added. One passes the team id, the input and output directories and a list of subjects (or asking for a random number of subjects) to start a pipeline:
`python narps_open/runner.py -t 2T6S -d /data/ds001734/ -o /output/ -s 001 006 020 100`
Furthermore:
    - :blue_book: documentation added in :file_folder: `docs`
    - :microscope: unit testing added in :file_folder: `tests`

### CODE REVIEW

**Comment by cmaumet** [comment]
Thank you @bclenet! This looks great! A quick question: can you point me to where the regression test is implemented? Thanks!

**Comment by bclenet** [comment]
The regression test is not in the PR as it is not implemented yet. I had to perform it locally and mainly by hand, because it is quite tricky to fully automatize for now.

Here is a howto:
- from the code revision to be tested (commit [420b499](https://github.com/Inria-Empenn/narps_open_pipelines/pull/24/commits/420b499a167f4f21969b9af6753d09b3f79753b7)), I ran the pipeline 2T6S with 4 subjects (001, 002, 003, 004) and stored the results of the 2nd level analysis (con* files and spmT* files)
- I switched to commit [c70e820](https://github.com/Inria-Empenn/narps_open_pipelines/commit/c70e82096e5667148bb1dbd49f1a823023b8adfe) and performed the same processing (2T6S with 4 subjects...)
- I then compared the files between each other using a Pearson correlation.

I can add a note/documentation explaining this, as well as the code for the correlation.

**Comment by cmaumet** [comment]
@bclenet -- adding documentation and code on how to do the regression test would be great!

**Comment by bclenet** [comment]
Hi @cmaumet, I just pushed the code & documentation for the non regression test.

File: docs/testing.md [hunk 1]
```diff
@@ -29,9 +29,250 @@ Tests can be launched manually or while using CI (Continuous Integration).
 * [CI] to output a xml JUnit test report, use the option : `--junit-xml=pytest_report.xml`
 * To create code coverage data : `coverage run -m pytest ./tests` then `coverage report` to see the code coverage result or `coverage xml` to output a .xml report file
 
-## Writing tests
-The main idea is to create one test file per source module (eg.: *tests/pipelines/test_pipelines.py* contains all the tests for the module `narps_open.pipelines`).
+## Writing unit tests
```
Comment (cmaumet) [inline docs/testing.md:32]: Reading this document, I've just noticed a small typo  "Dependancies" > "Dependencies"

**Comment by cmaumet** [comment]
Thanks @bclenet! This update looks good to me, let's merge!

**Pull request merged**